packages/cli/src/modules/chat-hub/chat-hub.service.ts

```diff
@@ -737,13 +899,13 @@ export class ChatHubService {
 			// TODO: We should consider can we just save the output from the captured stream always instead
 			// of parsing it from execution data, which seems error prone, especially with custom workflows.
 			// That could make handling multiple agents, multiple runes, tool executions etc easier...?
-			const output = this.getAIOutput(execution, NODE_NAMES.REPLY_AGENT);
-			if (!output) {
-				throw new OperationalError('No response generated');
-			}
+			// const output = this.getAIOutput(execution, NODE_NAMES.REPLY_AGENT);
```

Comment:
I'm also reworking how this works in cha-42.

Now it's just saving whatever was streamed (instead of reading it from a agent with hardcoded name, which would've been needed to change to find the agent node from WF on custom WFs), which seems to work fine.